### DISCUSSION TITLE
Welcome to PomodoroLogger Discussions!

### DISCUSSION THREAD

**Discussion by zxch3n** [zxch3n/PomodoroLogger#181] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by CaptainPackers** [comment]
Is there a way to archive cards on the Kanban board?

**Comment by CaptainPackers** [comment]
A user guide would be nice.

**Comment by VictoireWood** [comment]
Hope  there will be a ***Dark Mode*** Soon. 
And, hope the logger record in detail page won't **crash** when I put my pointer on the purple bar in the middle.

**Comment by s-jevtic** [comment]
It would be good to have an estimate on when a project will be done (not the time left). For example, something like: `estimated finish time: 23:32`.
